feat(components):  [autocomplete] `popper-class` support object

Heising opened the pull request.

Heising committed:
chore(components):  component data structure using object syntax

btea commented:
Hi, thanks for your contribution. It seems that supporting strings is sufficient for now, could you explain the reason for your change?

Heising commented:
> Hi, thanks for your contribution. It seems that supporting strings is sufficient for now, could you explain the reason for your change?

need to dynamically change the CSS of the dropdown menu, this is a feature that Vue already has, and you should not disable it just by using strings

Heising committed:
docs(components): [autocomplete] fetch-suggestions revise type 

type: definePropType<AutocompleteFetchSuggestions>([Function, Array])
source code support array type
